server/src/main/java/org/elasticsearch/index/codec/tsdb/TSDBDocValuesEncoder.java

```diff
@@ -176,7 +176,7 @@ private void forEncode(int token, int tokenBits, long[] in, DataOutput out) thro
      * Encode the given longs using a combination of delta-coding, GCD factorization and bit packing.
      */
     void encode(long[] in, DataOutput out) throws IOException {
-        assert in.length == numericBlockSize;
+        // assert in.length == numericBlockSize;
```

Comment:
Why are these asserts commented out?

Reply:
Sorry this is a msitake in merging main probably.